specs/kubernetes/list_events_for_a_single_kubernetes_pod.yaml

```diff
@@ -4,7 +4,13 @@ command: "kubectl get event --namespace {{namespace}} --field-selector involvedO
 tags:
   - kubernetes
 description: Lists the events for a single Kubernetes pod by using the field-selector flag.
-arguments: []
+arguments:
+  - name: namespace
+    description: ~
```

Comment:
Could we add descriptions here (and any other arguments we added) as well? Will make the UI better